UI/CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ find_package(Threads REQUIRED)
 find_package(Qt5Network ${FIND_MODE})
 find_package(Qt5Widgets ${FIND_MODE})
 find_package(Qt5Svg ${FIND_MODE})
-if(WIN32)
+if(WIN32 AND (Qt5Widgets_VERSION VERSION_GREATER_EQUAL 6.0.0))
```

Comment:
```suggestion
if(WIN32 AND (Qt5Widgets_VERSION VERSION_LESS 6.0.0))
```

Reply:
I really should not work in the midnight. Sorry that my brain isn't exactly working great.